.eslintrc.cjs

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  root: true,
+  env: {
+    es2021: true,
```

Comment:
The underlying engine quickjs only support es2020: https://bellard.org/quickjs/ so we need es2020 here

Reply:
@ailisp This is just for the code we are writing, doesn't have to do anything with the target for the TypeScript compilation. So we are using newer features in code, or at least we could be, but then the TypeScript compiler will build them to whatever version we choose in the `tsconfig.json` file.